oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -514,41 +514,37 @@ impl GrpcInvocationHandler {
 
         // The channel containing the request is created with the label specified by the caller.
         // This will fail if the label has a non-empty integrity component.
-        let (request_writer, request_reader) = self
-            .runtime
-            .channel_create_with_downgrade("gRPC request", &label)
-            .map_err(|err| {
-                warn!("could not create gRPC request channel: {:?}", err);
-            })?;
+        let (request_sender, request_receiver) =
+            channel_create_with_downgrade::<GrpcRequest>(&self.runtime, "gRPC request", &label)
+                .map_err(|err| {
+                    warn!("Couldn't create gRPC request channel: {:?}", err);
+                })?;
+
         // The channel containing the response is created with the label containing the identity of
         // the user.
-        let (response_writer, response_reader) = self
-            .runtime
-            .channel_create_with_downgrade("gRPC response", identity_label)
-            .map_err(|err| {
-                warn!("could not create gRPC response channel: {:?}", err);
-            })?;
+        let (response_sender, response_receiver) = channel_create_with_downgrade::<GrpcResponse>(
```

Comment:
Does it make sense to make a similar change for `HttpServerNode`? Channel creation is actually done in the `Pipe`:
https://github.com/project-oak/oak/blob/2bdae2477e5f0cb368b8cbbf6033dc8493e21a26/oak_runtime/src/node/http/util.rs#L59